app/src/main/java/com/bogdan/codeforceswatcher/fragment/ContestsFragment.kt

```diff
@@ -35,21 +33,21 @@ class ContestsFragment : android.support.v4.app.Fragment(), SwipeRefreshLayout.O
     }
 
     private fun initViews() {
-        swiperefreshContests.setOnRefreshListener(this)
+        swiperefresh.setOnRefreshListener(this)
```

Comment:
It's called Camel Case, when each new word in name is written from Uppercase letter. Here it will be `swipeToRefresh`.